### PR TITLE
[BIP-739]-Delegate-Gyroscope-GGU-to-DAO-Participants

### PR DESCRIPTION
**Pull request by Zen-Maxi**
Proposal: https://forum.balancer.fi/t/bip-xxx-delegate-gyroscope-ggu-to-dao-participants/6218/2

Gyroscope Vault: https://etherscan.io/address/0xA2321E23B3060e160195E138b62F8498546B0247#writeContract

### CODE REVIEW

**Comment by Zen-Maxi** [comment]
Pending PR for bal_addresses update: https://github.com/BalancerMaxis/bal_addresses/pull/536

**Comment by Xeonus** [comment]
@Zen-Maxi payload is wrong, need veballer.eth as delegate, not defilytica.eth!

File: BIPs/2024-W50/[BIP-XXX]-Delegate-Gyroscope-GGU-to-DAO-Participants.json [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      "contractMethod": {
+        "inputs": [
+          { "internalType": "address", "name": "_delegate", "type": "address" },
+          { "internalType": "uint256", "name": "_amount", "type": "uint256" }
+        ],
+        "name": "delegateVote",
+        "payable": false
+      },
+      "contractInputsValues": {
+        "_delegate": "0x99AfD53f807766A8B98400B0C785E500c041F32B",
```
Comment (Xeonus) [inline BIPs/2024-W50/[BIP-XXX]-Delegate-Gyroscope-GGU-to-DAO-Participants.json:44]: Change to veballer.eth, thanks

**Review by Xeonus — CHANGES_REQUESTED** [review]

**Review by Xeonus — APPROVED** [review]
Has correct signer set now, incl. 0x9f74662aD05840Ba35d111930501c617920dD68e - veballer.eth

**Pull request merged**